src/qt/guiutil.cpp

```diff
@@ -119,6 +120,14 @@ void setupAddressWidget(QValidatedLineEdit *widget, QWidget *parent)
     widget->setCheckValidator(new BitcoinAddressCheckValidator(parent));
 }
 
+void setButtonShortcuts(QPushButton* button, QWidget* parent, const QList<QKeySequence>& shortcuts)
+{
+    QAction* action = new QAction(parent);
+    action->setShortcuts(shortcuts);
+    parent->addAction(action);
+    parent->connect(action, &QAction::triggered, [button]() { button->animateClick(); });
```

Comment:
Why the `action` is added to the `parent`, not to the `button`?

Reply:
right, will fix

Reply:
addressed in  ab95506